src/datanode/src/sql/create.rs

```diff
@@ -135,6 +135,27 @@ impl SqlHandler {
             .map(|(k, v)| (v, k))
             .collect::<HashMap<_, _>>();
 
+        let pk_map = stmt
+            .columns
+            .iter()
+            .filter(|col| {
+                col.options
+                    .iter()
+                    .any(|options| match options.option.clone() {
+                        ColumnOption::Unique { is_primary } => is_primary,
+                        _ => false,
+                    })
+            })
+            .enumerate()
+            .map(|(_, col)| col.name.value.clone())
+            .collect::<Vec<_>>();
```

Comment:
I've tried SQL like 
```sql
CREATE TABLE test (
	a INTEGER PRIMARY KEY,
	b INTEGER PRIMARY KEY,
	c TEXT,
);
```
in MySQL, PostgreSQL and DuckDB, all of them don't allow declare multiple PRIMARY KEY constrain. You can only write them like
```sql
CREATE TABLE test (
	a INTEGER PRIMARY KEY,
	b INTEGER ,
	c TEXT,
);
```
or
```sql
CREATE TABLE test (
	a INTEGER ,
	b INTEGER ,
	c TEXT,
        PRIMARY KEY(a,b)
);
```

So maybe we should follow this rule, only allow one PK constrain here

Reply:
sure, I have added a throw when multiple inline pk found.